types/needle/needle-tests.ts

```diff
@@ -78,10 +78,21 @@ function API_get() {
             // if no http:// is found, Needle will automagically prepend it.
         });
 
```

Comment:
```suggestion
let resp: needle.NeedleResponse;
```

Reply:
I already declared `let resp: needle.NeedleResponse` here: https://github.com/Borewit/DefinitelyTyped/blob/8ceafbecc7a4c2e0d23227b4c6387ee3fa70d75d/types/needle/needle-tests.ts#L92

Do you want me to do something with the return value?

Reply:
I suggested moving the `resp`'s declaration here so you can demonstrate the `needle()` return type, in place of the whole `getResponse()` function.

You then will use `resp` at line 93.